server/server.js

```diff
@@ -29,15 +27,21 @@ app.use(bodyParser.urlencoded({ extended: true }));
 */
 
 const session = {
+  store: new PGSession({
+    pool: pgPool,
```

Comment:
are you supposed to be using the `pgPool` exported from the db file? Or the entire reference here? If the former, then I think you'll want to use `db.pgPool` (see comment above)

Reply:
We want to be using the pgPool exported from the db file.